earthpy/spatial.py

```diff
@@ -446,13 +446,28 @@ def hillshade(arr, azimuth=30, angle_altitude=30):
         >>> plt.imshow(shade) #doctest: +ELLIPSIS
         <matplotlib.image.AxesImage object at 0x...>
     """
-    azimuth = 360.0 - azimuth
+    try:
```

Comment:
@jlpalomino this ValueError message is a bit unclear. Will gradient fail is the array has more than 2 dimensions? i wonder if we can make the output error messsage more specific.

This tests to ensure it's a numpy array and than it is two dimensions? is that correct or do you have another intent here ?

Reply:
That is correct - the gradient will not compute successfully if the input array is not two-dimensional. I've updated the ValueError to be more clear ("Input array should be two-dimensional").